free-1.md

```diff
@@ -57,12 +67,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+It seems like it would be a 60-40 split, with 60 percent of the time spent directly engaging in the language aspects of the project and 40 percent spent on the systems aspect. As discussed in the scope question, I feel the most important part of this language is the user-friendly aspect. The project could easily be 99% systems in that the language will have several audio manipulating tools, but determining an effective language design feels more relevant considering my intentions.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Although the idea seems big and extremely ambitious, it can be rescoped to only include a few audio tools, with most of the project focusing on directly engaging with the langauge aspects of the project. The language design decisions are probably the most important aspect of this project as the language is built with the intention of making the process easier to understand nad less overwhelming.
 
```

Comment:
As I said above, even if you only do a few audio tools, be sure to know exactly what you want your users to be able to do with them. I think this is an awesome idea for a DSL, just be sure that it doesn't get confusing for users who want to mix multiple songs or make their own song. As a suggestion, it might be a good idea to prioritize user friendliness and mixing one song, then go from there.